.github/workflows/django.yml

```diff
@@ -79,6 +79,7 @@ jobs:
           isort --diff -c meinberlin tests
           python manage.py makemigrations --dry-run --check --noinput
           flake8 meinberlin tests --exclude migrations,settings
+          mkdir -p media
```

Comment:
so same as local, not sure on this but as it's needed for the local setup it should probably be in the test setup too right?